README.md

```diff
@@ -44,6 +45,10 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+<a id="About"></a>>
+## About
+nvm is a version manager for nodejs and npm. Each user will gets his/hers own versions of nodejs. Nvm works on linux and macOS, windows might work but is not guaranteed.
```

Comment:
- "nodejs" is only what ubuntu calls it; it's node.js or node
 - "theirs" instead of "his/hers"; gender isn't binary
 - `nvm` should always be lowercased
 - `nvm` is guaranteed to work on Windows WSL, since it's POSIX-compliant.